packages/viewer/src/components/scene-tree/lib/controller.ts

```diff
@@ -121,8 +122,15 @@ export class SceneTreeController {
    */
   public onStateChange = new EventDispatcher<SceneTreeState>();
 
+  /**
+   * A dispatcher that emits an event whenever the number of search results has changed.
+   */
+  public numberOfRowsWithFilterHitChanged = new EventDispatcher<number>();
```

Comment:
Looks like this value is contained within state. Can we just rely on the `onStateChange` dispatcher?